packages/create/src/generators/wc-lit-element-ts/templates/_tsconfig.json

```diff
@@ -0,0 +1,15 @@
+  {
+  "compilerOptions": {
+    "target": "esnext",
```

Comment:
We should pick a target here, `es2018` or `es2019`, need to see what works on latest browsers.